pkgs/development/guile-modules/guile-json/default.nix

```
@@ -0,0 +1,24 @@
+{ stdenv, fetchurl, guile, texinfo, pkg-config }:
+
+assert stdenv.cc.isGNU;
+
+stdenv.mkDerivation rec {
+  pname = "guile-json";
+  version = "0.3.0";
+
+  src = fetchurl {
+    url = "mirror://savannah/guile-json/${name}.tar.gz";
```

```suggestion
    url = "mirror://savannah/guile-json/${pname}.tar.gz";
```

Is this what it should be? Fails to eval because of this (ran nixpkgs-review locally to discover that).

---

An above review broke this, it still built locally because of hash, my bad